docs/src/components/context-menu.md

```
@@ -0,0 +1,68 @@
+            <VCContextMenuItem @select="onAction('delete')">Delete</VCContextMenuItem>
+        </VCContextMenuContent>
+    </VCContextMenu>
+</template>
+```
+
+## Theme keys
+
+| Key | Default class | Notes |
+|---|---|---|
+| `trigger` | `vc-context-menu-trigger` | |
+| `content` | `vc-context-menu-content` | Menu panel; `data-state="open|closed"` for animation. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)